src/eva/vision/models/modules/semantic_segmentation.py

```diff
@@ -22,9 +22,10 @@ class SemanticSegmentationModule(module.ModelModule):
 
     def __init__(
         self,
-        decoder: decoders.Decoder,
+        decoder: decoders.Decoder | nn.Module,
         criterion: Callable[..., torch.Tensor],
         encoder: Dict[str, Any] | Callable[[torch.Tensor], List[torch.Tensor]] | None = None,
+        freeze_encoder: bool = False,
```

Comment:
I believe that the encoder freezes when `lr_multiplier_encoder=0`. so we can escape an additional argument

Reply:
Oh right, missed that.